src/controllers/controller.bar.js

```diff
@@ -213,35 +213,33 @@ module.exports = function(Chart) {
 
 		updateElement: function(rectangle, index, reset) {
 			var me = this;
-			var chart = me.chart;
 			var meta = me.getMeta();
 			var dataset = me.getDataset();
-			var custom = rectangle.custom || {};
-			var rectangleOptions = chart.options.elements.rectangle;
+			var options = me._resolveElementOptions(rectangle, index);
 
 			rectangle._xScale = me.getScaleForId(meta.xAxisID);
 			rectangle._yScale = me.getScaleForId(meta.yAxisID);
 			rectangle._datasetIndex = me.index;
 			rectangle._index = index;
-
+			rectangle._options = options;
```

Comment:
This `_options` property looks to me to be unused

Reply:
I was anticipating making `hover*` options scriptable, where we need to keep options values into the element. I will remove it.